src/modules/numbers/dto/get-numbers-query.dto.ts

```diff
@@ -15,7 +15,7 @@ export class GetNumbersQueryDto {
 
   @IsString()
   @IsNotEmpty()
-  @ApiProperty({ example: '0030052431', description: 'UKEF ID to check' })
-  @Matches(/^\d{10}$/)
+  @ApiProperty({ example: '00300524', description: 'UKEF ID to check' })
```

Comment:
Might be prudent to add examples for:

1. Facility / Deal / Covenant ID: 0020019715
2. Party URN: 00300524

Reply:
![image](https://github.com/user-attachments/assets/f244801a-2695-4617-8558-b1dd4aeb9ef3)
good idea, this seems to be a good solution^

Reply:
Thanks